### DISCUSSION TITLE
Feature request: Comment block only mode

### DISCUSSION THREAD

**Discussion by webstory** [znck/grammarly#320] [Ideas]
Hi, I am a programmer and not an English native speaker.
I don't want a grammar check on an entire 'source code'(We can't change a function name in the library!) but only in the comment block like /* ... */ or // lines.

Thanks.

**Comment by klausbreyer** [comment]
this would be so helpful!

**Comment by EdoardoCostantini** [comment]
Same here! I would love the extension to allow me to check the spelling only for comments.

**Comment by bjerh** [comment]
Sometimes you need to add a comment to your code and simultaneously have it on point. This would be extremely helpful. I have been looking for this for ages.

**Comment by RasmusLeupold** [comment]
I would also much appreciate such a feature! 😄

It should be somewhat equally configurable than the `grammarly.files.include` setting option. But I guess it will be much more expensive to implement, even if it only works for single-line comments.
